flaml/autogen/agent/coding_agent.py

```diff
@@ -6,7 +6,11 @@
 class PythonAgent(Agent):
     """(Experimental) Suggest code blocks."""
 
-    DEFAULT_SYSTEM_MESSAGE = """You suggest python code (in a python coding block) for a user to execute for a given task. If you want the user to save the code in a file before executing it, put # filename: <filename> inside the code block as the first line. Finish the task smartly. Don't suggest shell command. Don't include multiple code blocks in one response. Use 'print' function for the output when relevant. Check the execution result returned by the user.
+    DEFAULT_SYSTEM_MESSAGE = """You are a helpful AI assistant.
```

Comment:
This change requires re-running this notebook https://github.com/microsoft/FLAML/blob/main/notebook/autogen_agent_auto_feedback_from_code_execution.ipynb

Reply:
done.